### PR TITLE
Add zenhub documentation

### PR DESCRIPTION
**Pull request by xmclark**
### Changes proposed within this pull request:
- Add zenhub documentation to Contributing.md
- Remove filler in Contributing.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-
-### Documentation
-- Links to documentation
+### ZenHub
+
+The software lifecycle in Toolbox is tracked using an external tool that sits on top of GitHub called ZenHub. ZenHub can be used from the [webapp](app.zenhub.com) or the [chrome extension](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd?hl=en-US). 
+
+ZenHub adds swim-lane boards which are used to organize issues. GitHub labels help organize issues in the swim-lanes and are required.  
+
+ZenHub also adds Milestones and Epics. Milestones are used for releases or features. Epics track larger development efforts which may span multiple releases.
```
Comment (DannyPeck) [inline .github/CONTRIBUTING.md]: Milestones are actually provided by Github in vanilla-Github.  I would still keep how we plan to use them in here though.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
+
+ZenHub is highly suggested for contributors because it allows one to see more information about issues, milestones, and epics that are not avaialble in vanilla-GitHub.
```
Comment (DannyPeck) [inline .github/CONTRIBUTING.md]: See milestone comment above.

**Review by DannyPeck — CHANGES_REQUESTED** [review]
Looks great!  Just one note about Milestones :)

**Comment by xmclark** [comment]
I've addressed the issue about the milestones. Will merge when codacy is done.

**Review by DannyPeck — APPROVED** [review]

**Pull request merged**